<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistrictProfileDialog</class>
 <widget class="QDialog" name="DistrictProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>District Profile</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>410</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>District Profile</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblDistrictName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>District Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDistrictNameValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblParentDistrict">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parent District:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblParentDistrictValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Area:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAreaValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHouseholds">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Households:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblHouseholdsValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblParcels">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcels:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblParcelsValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSchools">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Schools:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSchoolsValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPools">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pools:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPoolsValue">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>140</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DistrictProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>